refresher/consumer/engine/pulumiMapper.go

```diff
@@ -99,3 +111,13 @@ func PulumiMapper(
 	return nil
 
 }
+
```

Comment:
I know this is a but overhead but do you mind to move it to go-kit or even just put a TODO?